x11-plugins/pidgin-skypeweb/Manifest

```diff
@@ -3,3 +3,5 @@ DIST pidgin-skypeweb-1.2.2.tar.gz 410637 BLAKE2B 4766042f31159c3165e54267cd108d9
 DIST pidgin-skypeweb-1.2.tar.gz 323263 BLAKE2B 3a460fc53f53cb82608a9e09bfaa52c5ccd28350a485631725de953e020e9d05fb87a19e5e62198fe37e5590e763798ef8e20efd63ba84b46aeaf1ae314cb02a SHA512 12ff1d67e1eaaf075be49801031f7cd19351a40a599b7495bdb1003557defb72238cbba749d3eb18b5479df63291914a91a0b942ce5ba14f9362eca55428002c
 DIST pidgin-skypeweb-1.4.tar.gz 436177 BLAKE2B 1d78456a12d2ac52b7c0c6019148fad14ecb5a85f4d113dfa5c5916d64ecfaf6035b11cc348b8f3b0a7254a92ee0e35fef0fcacdb0ea69b092e528fab461728d SHA512 58a5bc0db5ddaeb3614366eecbb0d72f02e504bb10278a5df3075c8d2aa29e2f9cc18242616d8380044982a76440f51bdceec20b277d826e8b3066b1e303145e
 DIST pidgin-skypeweb-1.5.tar.gz 295956 BLAKE2B 0a45c709570c9093df9f0e674e5556a21b832b5e73ab2f32d439aa4fe3ce224df0bab05e054ef352ed337113e6800020cf0da60a7b37a6f39a22fc95ad40d28b SHA512 93043b83132654144f333e90bdff88dec5efe2495412a0b40f0ab64ec96d9427acea553ff73dee054de87e77121cbfd1a7bcaddeb17ab4d91baef55d658b6478
+DIST pidgin-skypeweb-1.6.tar.gz 296918 BLAKE2B a2be2d9b161d97db133b3da170be2cc0b5fbe0c39d1b8d5be34e838c3f106bebaf520bb45cd90e7d0315275af9a620190bd41e675a4626cdc4535660a3a344e9 SHA512 215894937f85f254718ab46d7b817d8fc04893d967305b3617b53c724a8c0b616df5fbe0e2036097b9dff5d7dc7e183174ab6d1e3994735469be67d3c0c1c012
```

Comment:
There's no reason to add 1.6, since it will have matching KEYWORDS to 1.7 and no one will ever install it.